content/security/advisory/2018-06-25.adoc

```diff
@@ -38,7 +38,7 @@ issues:
       fixed: 1.14
       previous: 1.13
   description: |
-    SSH Credentials Plugin allowed the creation of SSH credentials with keys "From a file on Jenkins master".
+    SSH Credentials Plugin allowed the creation of SSH credentials with keys "From a file on Jenkins controller".
```

Comment:
This refers to a UI label that is now incorrectly identified.

Reply:
Deprecated anyway: https://github.com/jenkinsci/ssh-slaves-plugin/blob/dbbf834c07ab5318c3b29f070c0108c93f5596a7/doc/TROUBLESHOOTING.md#after-upgrade-to-ssh-slaves-128-failed-to-connect-using-ssh-key-credentials-from-files

Reply:
Well yeah, this advisory removed it. Doesn't make the change any more correct, in fact it's unlikely to get changed in the plugin now ;-)

Reply:
doh, I totally missed that. I'll be more careful in the future.